src/server/game/Entities/Player/PlayerStorage.cpp

```diff
@@ -7206,13 +7206,21 @@ void Player::_SaveAuras(CharacterDatabaseTransaction trans, bool logout)
     stmt->SetData(0, GetGUID().GetCounter());
     trans->Append(stmt);
 
+    // Add here the auras IDs to exclude
+    std::unordered_set<uint32> excludedAuraIds = {42533};
```

Comment:
And dont hardcode things

Reply:
There’s a spell attribute to skip saving auras afaik